content/blog/writing-crossguard-policies/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Writing CrossGuard Policies"
+date: 2019-12-19
+meta_desc: "A look at the different types of policies that can be written for Pulumi CrossGuard."
+meta_image: crossguard.png
+authors:
+    - justin-vanpatten
+tags:
+    - crossguard
+    - policy-as-code
+---
+
+We recently announced [a new policy as code solution, CrossGuard]({{< relref "/blog/announcing-crossguard-preview" >}}) that validates policies at deployment time. Policies are expressed as code and are used to prevent the creation of out-of-compliance resources. Since policies are written using full-blown programming languages, it's possible to do interesting things such as [combining IAM Access Analyzer and Pulumi CrossGuard]({{< relref "/blog/aws-iam-access-analyzer-and-crossguard" >}}). In this post, we'll take a closer look at the different types of policies that can be written.
```

Comment:
Seems like we should add a "what's in it for you" type sentence in the middle of the paragraph, e.g.:

... used to prevent the creation of out-of-compliance resources. [This allows an organization to prevent entire classes of security and reliability defects to ensure infrastructure is following best practices.] Since policies are written...